darshan-util/darshan-logutils.c

```diff
@@ -2207,6 +2207,11 @@ int darshan_log_get_record(darshan_fd fd,
     return r;
 }
 
+void darshan_free(void *ptr)
```

Comment:
maybe a comment here (or in the header, whatever is most like the rest of the code) to indicate that this mostly exists as a helper routine for the python bindings in case anyone sees it in the C API and wonders what it does.

Reply:
I added a short explanation above the function, following the convention in this file. The header file doesn't seem to have comments for the functions in this source file.

Reply:
Great, thanks!